main.go

```diff
@@ -1,5 +1,94 @@
 package main
 
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/reconciler"
+
+	"github.com/jackc/pgconn"
+	"go.uber.org/zap"
+)
 
 func main() {
+	var logger *zap.Logger
+	var err error
+	if os.Getenv("DEVELOPMENT") != "" {
+		logger, err = zap.NewDevelopment()
+	} else {
+		logger, err = zap.NewProduction()
+	}
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx := context.Background()
+	// parse configuration from yaml file
+	conf, err := db.GetConfig(logger, "config/config.yaml")
+	if err != nil {
+		logger.Fatal(err.Error())
+	}
```

Comment:
An example of local error variables. Please apply to all relevant statements below.
```suggestion
	if conf, err := db.GetConfig(logger, "config/config.yaml"); err != nil {
		logger.Fatal(err.Error())
	}
```

Reply:
we cannot access the conf as it can only be accessed inside the condition

Reply:
You can predefine it in the outer scope. The issue you see here is shadowing